server/index.js

```diff
@@ -184,6 +196,32 @@ app.get("/files/:filename", (req, res) => {
   });
 });
 
+app.post("/files/likes", (req, res) => {
```

Comment:
In the future, it would be best to place these routers and methods within their own router pages, since index.js should routes from the router document, rather than having all routes posted together in index.js.